### PR TITLE
Support python 3.7, 3.8 in conditional section expressions

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Review by mgedmin — APPROVED** [review]
:100:

It would be great if you could add a CHANGES.rst note before merging.  Please mention something like "support `python37` and `python38` in conditional section expressions", rather than a more generic Python 3.7/3.8 support which I think was already added (and mentioned) before.

**Comment by mauritsvanrees** [comment]
Might as well add `python39` I would say: 3.9.0 is expected early next month.

**Comment by gotcha** [comment]
@hbrunn Can you rebase on latest master where your fix for tests has been merged ?

**Comment by hbrunn** [comment]
@gotcha done, I also took @mauritsvanrees' advise to include 3.9 immediately

**Comment by gotcha** [comment]
@hbrunn Thanks ! Good for 3.9 !

**Comment by mauritsvanrees** [comment]
Minor detail: we should include `python39` in the `CHANGES.rst` note, but that could be done directly on master after merge, if Travis is green.

**Comment by gotcha** [comment]
Thanks for finding this.

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Change History
 3.0.0a3 (unreleased)
 ====================
 
-- Nothing changed yet.
+- Support python37 and python38 in conditional section expressions
```
Comment (icemac) [inline CHANGES.rst:7]:
```suggestion
- Support python37, python38 and python39 in conditional section expressions
```

**Comment by icemac** [comment]
I think this PR should be ready to merge now.

**Review by icemac — APPROVED** [review]

**Comment by mgedmin** [comment]
Why doesn't GH actions let you re-run just one job?

**Comment by icemac** [comment]
> Why doesn't GH actions let you re-run just one job?

So the only way to get all jobs green is to burn a lot of energy. —  Is there a GitHub issue tracker to report this?

**Comment by mgedmin** [comment]
> Is there a GitHub issue tracker to report this?

[Famously not](https://blog.thea.codes/we-need-to-talk-about-github/).

**Pull request merged**